camel-openmrs-fhir/src/main/java/org/openmrs/eip/fhir/Constants.java

```diff
@@ -22,6 +22,12 @@ public final class Constants {
 	
 	public static final String TEST_ORDER_TYPE_UUID = "52a447d3-a64a-11e3-9aeb-50e549534c5e";
 	
+	public static final String IMAGING_ORDER_TYPE_UUID = "8d2aff07-55e6-4a4a-8878-72b9eb36a3b8";
+	
+	public static final String PROCEDURE_ORDER_TYPE_UUID = "67a92e56-0f88-11ea-8d71-362b9e155667";
+	
+	public static final String SUPPLY_REQUEST_ORDER_TYPE_UUID = "67a92bd6-0f88-11ea-8d71-362b9e155667";
+	
 	public static final String DRUG_ORDER_TYPE_UUID = "131168f4-15f5-102d-96e4-000c29c2a5d7";
 	
```

Comment:
Generally, hardcoding UUIDs is not ideal, as different implementations may require different sets of UUIDs. I would prefer making these configurable with default values, rather than embedding them directly into the code.

Reply:
Yeah you are right. But let's take this up separately as this is an existing practice for this repo.

Have created a ticket [here](https://mekomsolutions.atlassian.net/browse/OZ-812)

Reply:
Unfortunately, this issue is becoming more prominent as it is going to be consumed by existing distributions with pre-existing data. The sooner this is addressed, the better. I believe this can be resolved in the current PR, as it already includes the majority of the referenced UUIDs.

Reply:
@Ruhanga I would not want to include the changes you suggested in this PR. I have created a [new PR](https://github.com/openmrs/openmrs-eip/pull/69/commits/e1f0d3ad291710b90a21ae10e0f2ceb992c7242e) for the above ticket. Will need to fix test cases to make it work.

I would request @rbuisson to please merge this PR as this is blocking HSC deployment.